.github/release-drafter.yml

```diff
@@ -0,0 +1,28 @@
+
+name-template: 'Prebid $RESOLVED_VERSION Release'
+tag-template: '$RESOLVED_VERSION'
+categories:
+  - title: '🚀 New Features'
+    label: 'feature'
+  - title: '🧰 Maintenance'
```

Comment:
for consistency with old release notes, should we use 🛠? I personally like the icon you picked, but just wondering if we care about keeping it the same or not

Reply:
I just missed that. My goal was to match. I'll update it.